<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-22T22:07:14 IST" name="Default suite" finished-at="2025-02-22T22:07:15 IST" duration-ms="1503">
    <groups>
    </groups>
    <test started-at="2025-02-22T22:07:14 IST" name="Default test" finished-at="2025-02-22T22:07:15 IST" duration-ms="1503">
      <class name="test.test.SeleniumSubmitOrder">
        <test-method is-config="true" signature="LaunchApplication()[pri:0, instance:test.test.SeleniumSubmitOrder@6aeb35e6]" started-at="2025-02-22T22:07:14 IST" name="LaunchApplication" finished-at="2025-02-22T22:07:15 IST" duration-ms="1427" status="FAIL">
          <exception class="io.github.bonigarcia.wdm.config.WebDriverManagerException">
            <message>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1422)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1203)
at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:1438)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1415)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1203)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:418)
at test.TestComponents.BaseTest.initializeDriver(BaseTest.java:25)
at test.TestComponents.BaseTest.LaunchApplication(BaseTest.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux
at io.github.bonigarcia.wdm.online.UrlHandler.getCandidateUrl(UrlHandler.java:287)
at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:1279)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1197)
... 36 more
]]>
            </full-stacktrace>
          </exception> <!-- io.github.bonigarcia.wdm.config.WebDriverManagerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
        <test-method signature="SeleniumSubmitOrder()[pri:0, instance:test.test.SeleniumSubmitOrder@6aeb35e6]" started-at="2025-02-22T22:07:15 IST" name="SeleniumSubmitOrder" finished-at="2025-02-22T22:07:15 IST" duration-ms="0" status="SKIP">
          <exception class="io.github.bonigarcia.wdm.config.WebDriverManagerException">
            <message>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.github.bonigarcia.wdm.config.WebDriverManagerException: io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1422)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1203)
at io.github.bonigarcia.wdm.WebDriverManager.fallback(WebDriverManager.java:1438)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:1415)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1203)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:418)
at test.TestComponents.BaseTest.initializeDriver(BaseTest.java:25)
at test.TestComponents.BaseTest.LaunchApplication(BaseTest.java:52)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: io.github.bonigarcia.wdm.config.WebDriverManagerException: No proper candidate URL to download geckodriver 0.35.0-linux
at io.github.bonigarcia.wdm.online.UrlHandler.getCandidateUrl(UrlHandler.java:287)
at io.github.bonigarcia.wdm.WebDriverManager.download(WebDriverManager.java:1279)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:1197)
... 36 more
]]>
            </full-stacktrace>
          </exception> <!-- io.github.bonigarcia.wdm.config.WebDriverManagerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SeleniumSubmitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:test.test.SeleniumSubmitOrder@6aeb35e6]" started-at="2025-02-22T22:07:15 IST" name="tearDown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1740242235524" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.test.SeleniumSubmitOrder -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
